third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -167,6 +167,6 @@ private void deleteAllUsersDocuments() {
     }
 
     public String getTechnicalAccountUUID() {
-        return technicalAccountUUID;
+        return this.technicalAccountUUID;
```

Comment:
Why do you need to prefix the field with `this` ?

Reply:
I think it make more clear context.